### PR TITLE
Ensure that the AWS version is always ending with vx.x.

### PR DESCRIPTION
**Pull request by TurboGit**
This makes this test more stable across all version naming.

For RC11-007 & U726-036.

### CODE REVIEW

File: regtests/0192_client_headers/client_headers.adb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ------------------------------------------------------------------------------
 --                              Ada Web Server                              --
 --                                                                          --
---                     Copyright (C) 2008-2017, AdaCore                     --
+--                     Copyright (C) 2008-2018, AdaCore                     --
```
Comment (anisimkov) [inline regtests/0192_client_headers/client_headers.adb:4]: 2018 ?

**Review by anisimkov — CHANGES_REQUESTED** [review]

**Comment by TurboGit** [comment]
It's a cherry pick (as-is) of another commit from 2018.

File: regtests/0192_client_headers/client_headers.adb (hunk 1 above)
Comment (TurboGit) [reply regtests/0192_client_headers/client_headers.adb:4]: It's a cherry pick (as-is) of another commit from 2018.

**Review by anisimkov — APPROVED** [review]
I think we should cherry-pick it into all branches, at least into master to do not repeat it on each new LTS release.

**Comment by anisimkov** [comment]
> I think we should cherry-pick it into all branches, at least into master to do not repeat it on each new LTS release.

I see the master already have it.

**Comment by TurboGit** [comment]
Exactly : eeb5dedd3fd

**Pull request merged**